message_view.py

```diff
@@ -0,0 +1,20 @@
+import sublime
+import sublime_plugin
+
+PANEL_NAME = "SublimeLinter Messages"
+
+
+class SublimeLinterDisplayPanelCommand(sublime_plugin.TextCommand):
+    def run(self, edit, msg=""):
+        panel_view = sublime.active_window().create_output_panel(PANEL_NAME, True)
+        panel_view.set_read_only(False)
+        panel_view.erase(edit, sublime.Region(0, panel_view.size()))
+        panel_view.insert(edit, 0, msg)
+        panel_view.set_read_only(True)
+        panel_view.show(0)
+        sublime.active_window().run_command("show_panel", {"panel": "output.{}".format(PANEL_NAME)})
+
+
+class SublimeLinterRemovePanelCommand(sublime_plugin.TextCommand):
+    def run(self, edit):
+        sublime.active_window().destroy_output_panel(PANEL_NAME)
```

Comment:
Why does this need its own command? For symmetry with `sublime_linter_display_panel`? 

And why does it not use `self.view.window()` (and why is it not a `WindowCommand`).

Reply:
For symmetry indeed. Otherwise the caller would have to know the PANEL_NAME, we'd have to share that in some way, etc. I thought this was a bit more elegant.

The creation command needs to be a TextCommand I think. I borrowed it from [GitSavvy](https://github.com/divmain/GitSavvy/blob/bee6b548b5e66bb85b3bde7f7d740f7be318c5f2/common/commands/log.py). It does an erase and insert, which I don't think you can do via a WindowCommand?

The removal command could perhaps be a WindowCommand. I just set them up in a similar way and it worked. Your proposals don't work so I must be missing something.

Reply:
Iirc output panels are cleared when "create_output_panel" is called and the insertion could easily be done wit the append command, but I'm.not entirely sure on the first one.

Reply:
Ok, sounds good. I'll look into it some more.